ccd-definition/CaseEventToFields/response/partaccept/DefendantBankAccounts.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
```

Comment:
Please could you change the  remove the package name 'partaccept' and place it directly under CaseEventToFields. The screens may be shared across journeys and work streams, so it would allow to reuse where we can